src/Compilers/CSharp/Test/Emit/PDB/CSharpPDBTestBase.cs

```diff
@@ -17,6 +20,64 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.PDB
 {
     public class CSharpPDBTestBase : CSharpTestBase
     {
+        public static void TestLambdasAndClosures(string input, string methodStartSpanName = "method", string closureSpanName = "closure", string lambdaSpanName = "lambda")
+        {
+            MarkupTestFile.GetSpans(input, out var source, out IDictionary<string, ImmutableArray<TextSpan>> spans);
+
+            var compilation = CreateCompilationWithMscorlib40AndSystemCore(source, options: TestOptions.DebugDll);
+            compilation.GetDiagnostics().Where(d => d.Severity == DiagnosticSeverity.Error).Verify();
+
+            var pdb = PdbValidation.GetPdbXml(compilation);
+
+            //TODO: Should we support having more than one method under test? Or does that make for tests that do too much?
+            var methodStart = spans[methodStartSpanName].Single().Start;
```

Comment:
> var methodStart = spans[methodStartSpanName].Single().Start; [](start = 11, length = 61)

Report better message (assertion) when there is no method starting span or there is more of them.